proxy/coco/tests/gossip.rs

```diff
@@ -174,9 +177,24 @@ async fn can_ask_and_clone_project() -> Result<(), Box<dyn std::error::Error>> {
 
     requested(query_listener, &urn).await?;
     assert_cloned(clone_listener, &urn.clone(), alice_peer_id).await?;
-    // TODO(finto): List projects
-    let project = state::get_project(&bob_peer, urn).await;
+    let project = state::get_project(&bob_peer, urn.clone()).await;
     assert!(project.is_ok());
 
+    let announced = async_stream::stream! { loop { yield alice_events.recv().await } }
```

Comment:
It might make more sense for `alice` to `track` `bob` before he requests, to line up with the case we were testing for. And then also assert that `alice` has `bob` as a remote of the project.

Reply:
But that's not really what we test here. The thing that is asserted is that there will be a gossip message after a project has been successfully cloned through the waiting room.

Reply:
Then I would say it would be a nice _addition_ because that's the behaviour we were trying to test out and confirm.